modules/ROOT/pages/clauses/delete.adoc

```diff
@@ -53,6 +54,28 @@ This query is only possible to run on nodes without any relationships connected
 Deleted 1 node
 ----
 
+=== NODETACH keyword
+
+_This feature was introduced in Neo4j 5.14._
+
+It is also possible to delete the single node using the `NODETACH DELETE` clause.
+This behaves the same as omitting it.
```

Comment:
This is very tricky, and I think we need to be more explicit (at least until we can mention GQL, which will make documenting these features so much simpler)

```suggestion
Using the `NODETACH` keyword explicitly defines that relationships will not be detached and deleted from a node.
The `NODETACH` keyword is a mirror of the already existing keyword xref:clauses/delete.adoc#delete-a-node-with-all-its-relationships[DETACH], 
Including it is functionally the same as using simple `DELETE`.
```